cypress/unit/store-mutation-createColumnToCategoryMap.cy.js

```diff
@@ -0,0 +1,38 @@
+import { getters, mutations } from "~/store/index-refactor";
+
+var store = {};
+
+describe("createColumnToCategoryMap", () => {
+
+    beforeEach(() => {
+
+        store = {
```

Comment:
You should rename this (and the initial declaration above) to `state`, because that's what you are mocking here. Alternatively you could be explicit and mock `store.state`. We haven't done that in the tests so far because all they ever needede access to was the `state`. But I'm wondering if we should go back and change that, because it is a bit confusing.

Reply:
>We haven't done that in the tests so far because all they ever needed access to was the state. But I'm wondering if we should go back and change that, because it is a bit confusing.

Let's keep it for now, but I have created an issue for this [here](https://github.com/neurobagel/annotation_tool/issues/295).